ggml-cuda.cu

```diff
@@ -1,11 +1,12 @@
 #include <algorithm>
+#include <assert.h>
```

Comment:
Shouldn't this be `#include <cassert>`?

Reply:
Yes, and it's not the only one, but all I did here is sort the includes alphabetically.